conans/client/remote_manager.py

```diff
@@ -38,6 +38,15 @@ def __init__(self, cache, auth_manager, output, hook_manager):
         self._auth_manager = auth_manager
         self._hook_manager = hook_manager
 
+    def users_list(self, remotes):
```

Comment:
Same. Every call to ``RemoteManager`` should end in at least an API call to the remote. This is not the case, so please don't move this here.